src/binaryManager.ts

```diff
@@ -19,37 +21,42 @@ function getExtensionMirrordPath(): Uri {
 
 /**
  * Tries to find local mirrord in path or in extension storage.
+ * @param version If specified, then the version of the binary is checked and matched path is returned if it matches.
+ * @returns Path to mirrord binary or null if not found
  */
 export async function getLocalMirrordBinary(version?: string): Promise<string | null> {
     try {
-        const mirrordPath = await which("mirrord");
+        const mirrordPath = getExtensionMirrordPath();
```

Comment:
Why'd you switch around the order of the code in this function, i.e. checking in extension storage first?

Reply:
check extension storage first so that if we downloaded a binary we use that first and not in path

Reply:
Why wasn't this the order before? What changed?

Reply:
I am not sure why, i believe we should check extension path first

Reply:
Why'd the person who wrote the original code believe otherwise? Ping them and make sure there's nothing we're overlooking

Reply:
I believe we were originally checking `PATH` before the extension storage for our own convenience (make changes in the CLI -> add to `PATH` -> works). Given that we always check the version of binaries we find and that now we have other means to use preferred binary (e.g. by setting binary path in extension settings), I don't think the order matters at all